src/tracks/playabletrack/notetrack/ui/NoteTrackAffordanceControls.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
```

Comment:
We aren't yet wholly consistent about comment at tops of files, but see some examples with my name and imitate with your name in this and other new files.